### ISSUE TITLE
Signed url with convolution filter doesn't work. 

### ISSUE THREAD

**Issue by terrycheung** [thumbor/thumbor#514]
Hi,

I've been using thumbor-url to sign urls and thumbor throws a malformed URL error when i use a signed url that uses a convolution filter. Other filters such as brightness, upscale seem to work fine.

Any ideas? 
Thanks.

**Comment by guilhermef** [comment]
@terrycheung can you share an example ?

**Comment by terrycheung** [comment]
Using Thumbor 4.12.2 (was the latest at the time I set things up). Have created a scenario which replicates the issue.

Using the source image,
`http://images.huffingtonpost.com/2014-07-25-yahoologo.png`

Using the below secret key,
`mysecretkey`

The generated secure URL for thumbor is,
`http://thumbor/N2RdEjk8OGfCk-a0RyQmYG49KUk=/filters:convolution(1;2;1;2;4;2;1;2;1,3,true)/http://images.huffingtonpost.com/2014-07-25-yahoologo.png`

The result of using the above is that I get a 404 response from Thumbor. I would have expected 503 if the convolution filter was wrong. This works for all other filters I have tried. As previously mentioned check when checking the logs in debug mode I am getting a malformed URL error.
